src/Ast.ml

```diff
@@ -1703,6 +1705,12 @@ end = struct
     register_reset (fun () -> Hashtbl.clear memo) ;
     memo
 
+  let rec exposed_left_exp e =
+    match e.pexp_desc with
+    | Pexp_apply (op, _) when is_prefix op -> true
```

Comment:
Do we need a recursive call on `op` here ?

Reply:
It looks like it, though maybe there is some way that such parsetree values are never created. Recursing can't hurt I think.